### ISSUE TITLE
Publish Grafana Dashboard

### ISSUE THREAD

**Issue by prochac** [dataddo/pgq#18]
We could provide Grafana Dashboard, so everybody doesn't have to reinvent the wheel.

https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards/

Possibly in a separate repository?

**Comment by kedlas** [comment]
Good idea. However there must be the working data source in grafana to provide it with data. In our case in Dataddo we cannot access pgq db directly from Grafana because of security reasons and we use prometheus postgres exporter to provide the metrics which we later use as the source for data.

I will have a look on this one. Creating the dashboard with data source directly looking to the pgq should be quite straightforward.

**Comment by prochac** [comment]
True.
And now I remembered these issues with the exporter we use(?)

https://github.com/prometheus-community/postgres_exporter#adding-new-metrics-via-a-config-file-deprecated

https://github.com/prometheus-community/postgres_exporter/pull/801

**Comment by kedlas** [comment]
AFAIK now we have no issues now and everything works as expected with the exporter we use (https://github.com/helm/charts/blob/master/stable/prometheus-postgres-exporter/README.md)
